src/BuildScriptGenerator/Node/NodeBashBuildSnippet.sh.tpl

```diff
@@ -184,6 +186,7 @@ cd "$SOURCE_DIR"
 	echo "Switching to package directory provided: '{{ PackageDirectory }}'..."
 	if [ ! -d "$SOURCE_DIR/$packageDirName" ]; then
 		echo "Package directory '$SOURCE_DIR/$packageDirName' does not exist." 1>&2
```

Comment:
Do we need this `echo` call anymore? The `LogWarning` call is providing the same text to the user with additional information, so I think we should be safe to remove this to avoid redundancy.

Reply:
yah I was thinking the same. But kept it as it was already there.